src/redux/books/NavBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <div className="navBar">
+      <div className="navList">
+        <h1>Bookstore CMS</h1>
+        <p>
+          <Link className="link" to="/"> BOOKS </Link>
+        </p>
+        <p>
+          <Link className="link" to="/Category"> CATEGORIES </Link>
```

Comment:
- [ ] Please add the correct font family to the site logo and nav links because it's supposed to be `Montserrat` to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/200566738-2b0a242b-8b50-4949-b9b7-4fb0702e26bf.png)  | ![image](https://user-images.githubusercontent.com/68796232/200566676-e7764fa7-d831-4f7f-9f3c-757535c8d85b.png)  |